src/main/resources/db/migration/v0.39.0.3__agm_name_synonym_slot_annotations.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO slotannotation (id, slotannotationtype, singleagm_id, displaytext, formattext, nametype_id)
+  	WITH
+  		t1 AS (
+			SELECT id, name FROM affectedgenomicmodel WHERE name IS NOT NULL
+		),
+		t2 AS (
+			SELECT id FROM vocabularyterm WHERE name = 'full_name'
+		)
+	SELECT nextval('slotannotation_seq'), 'AgmFullNameSlotAnnotation', t1.id, t1.name, t1.name, t2.id
+	FROM t1, t2;
+	
+ALTER TABLE affectedgenomicmodel DROP COLUMN name;
+
+DROP TABLE affectedgenomicmodel_synonyms;
```

Comment:
@markquintontulloch do you know about how many this is?

Reply:
about 500,000 AGMs I think - didn't pay attention to how long it took locally, but not long enough to be of concern